packages/cli/src/eventbus/MessageEventBus/recoverEvents.ts

```diff
@@ -10,6 +10,7 @@ import { workflowExecutionCompleted } from '../../events/WorkflowStatistics';
 import { eventBus } from './MessageEventBus';
 import { Container } from 'typedi';
 import { InternalHooks } from '@/InternalHooks';
+import { WorkflowExecuteAdditionalData } from '../..';
```

Comment:
Can you please change this to `@/WorkflowExecuteAdditionalData`
```suggestion
import { WorkflowExecuteAdditionalData } from '@/WorkflowExecuteAdditionalData';
```

and remove the export for `WorkflowExecuteAdditionalData` from `src/index.ts`. This creates unnecessary circular dependencies.

Reply:
It'd have to be `import * as WorkflowExecuteAdditionalData from '@/WorkflowExecuteAdditionalData';` and it would not fix the cycle issue

Reply:
yeah. `WorkflowExecuteAdditionalData` is one of those files, alongside `WebhookHelpers` and `WorkflowHelpers` 😞 .
But this way the dependency "circle" is smaller.